src/GUI/ErikEJ.EntityFrameworkCore.SqlServer.Dacpac/DacPacEngine/DacpacTableListBuilder.cs

```diff
@@ -34,18 +34,24 @@ public DacpacTableListBuilder(string dacpacPath)
             using (var model = new TSqlTypedModel(dacpacPath))
             {
                 var result = model.GetObjects<TSqlTable>(DacQueryScopes.UserDefined)
-                    .Select(m => new Tuple<string, bool, List<string>>($"[{m.Name.Parts[0]}].[{m.Name.Parts[1]}]",
+                    .Select(m => new Tuple<string, bool, List<string>, List<string>>(
+
+                    $"[{m.Name.Parts[0]}].[{m.Name.Parts[1]}]",
+                    
                     m.PrimaryKeyConstraints.Any(),
```

Comment:
doesn't it throw in case on no PK? AFAIK, in case of view or table without PK, PrimaryKeyConstraints can be null? A couple of lines later you check for nullability.